src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/AbstractJavadocCheck.java

```diff
@@ -274,7 +300,20 @@ public final void visitToken(DetailAST blockCommentNode) {
             }
 
             if (result.getParseErrorMessage() == null) {
-                processTree(result.getTree());
+                final DetailNode javadocTree = result.getTree();
+                if (doesCheckAcceptJavadocWithNonTightHtml()) {
+                    processTree(javadocTree);
+                }
+                else {
+                    final DetailNode nonTightHtmlTag = JavadocUtils.getNonTightHtmlTag(javadocTree);
+                    if (nonTightHtmlTag == null) {
+                        processTree(javadocTree);
+                    }
+                    else if (doesCheckViolateJavadocWithNonTightHtml()) {
+                        log(nonTightHtmlTag.getLineNumber(), nonTightHtmlTag.getColumnNumber(),
+                                "javadoc.unclosedHtml", nonTightHtmlTag.getText());
```

Comment:
This property already existed in the message.properties, I didn't add it, should I update it ? Also, making it a public static final field results in errror from `XdocsPagesTest`, [XdocsPagesTest.java#L776](https://github.com/checkstyle/checkstyle/blob/master/src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsPagesTest.java#L776) , UT about missing error in other subclasses of `AbstractJavadocCheck`.